WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -330,6 +316,13 @@ on_retreat = {
 
 #Chinese traits
 son_of_heaven = {
+	potential = {
+		OR = {
+			culture_group = sinic
+			religion = confucian
+		}
+		higher_real_tier_than = KING
+	}
```

Comment:
The original files it came from were "VIET_ZHONGGOGU" or something like that. I searched for the Events they referenced, and they did not exist. Apparently, you're missed them when you isntalled VIET.